training/finetune_acft_hf_nlp/environments/acpt/spec.yaml

```diff
@@ -0,0 +1,13 @@
+$schema: https://azuremlschemas.azureedge.net/latest/environment.schema.json
+
+description: >-
+  Environment used by Hugging Face NLP Finetune components
+
+name: "{{asset.name}}"
+version: "{{asset.version}}"
+
+build:
+  path: "{{image.context.path}}"
+  dockerfile_path: "{{image.dockerfile.path}}"
+
+os_type: linux
```

Comment:
Could you please add preview tag for the components as well as environments?

Reply:
Like this: 

![image](https://user-images.githubusercontent.com/482709/229209056-337c9180-0c80-4df7-b42c-91e0e366dee5.png)